domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/NoticeExporter.java

```diff
@@ -0,0 +1,58 @@
+package org.mobilitydata.gtfsvalidator.domain.entity;
```

Comment:
This file should reside in notice package

Reply:
File has been moved in f8aebbedbac0edf1e24c10bca243a15d4edd66b5